packages/react-app/src/modules/Counter/counter.utils.js

```diff
@@ -0,0 +1,7 @@
+import { path, pipe, last, split } from 'ramda'
+
+export const getCounterIdFromPayload = payload => pipe(
```

Comment:
I think this should be here:
```
react-app/src/common/router/utils.js
```
and should be called:
```
getLastParamFromRouterState
```
and exposed in and index file in that module ^

also please remove payload var from the function, so it's headless:
```
export const getLastParamFromRouterState = pipe(
  path(['location', 'pathname']),
  split('/'),
  last,
)